network.sh

```diff
@@ -111,6 +111,11 @@ else
   exit 1
 fi
 
+function openshift_sdn_deprecated() {
+  # OpenShiftSDN is deprecated in 4.15 and later
+  printf '4.15\n%s\n' "$(openshift_version)" | sort -V -C
```

Comment:
Doesn't `function openshift_version()` need to be included in network.sh for this to work?

Reply:
Maybe its related to the ordering of the sources in` agent/06_agent_create_cluster.sh`, I had to define `openshift_version()` in network.sh in order to successfully run `make agent`

Reply:
Recommend moving `openshift_release_version()` to common.sh, that should take care of the ordering issue.